modules/300-prometheus/docs/FAQ.md

```diff
@@ -122,9 +122,34 @@ spec:
 ## How do I enable secure access to metrics?
 To enable secure access to metrics, we strongly recommend using **kube-rbac-proxy**.
 
+## How do I add Alertmanager?
+
+Create a Custom Resource `CustomAlertmanager` with type `Internal`.
+Example:
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: CustomAlertmanager
+metadata:
+  name: webhook
+spec:
+  type: Internal
+  internal:
+    route:
+      groupBy: ['job']
+      groupWait: 30s
+      groupInterval: 5m
+      repeatInterval: 12h
+      receiver: 'webhook'
+    receivers:
+    - name: 'webhook'
+      webhookConfigs:
+      - url: 'http://webhookserver:8080/'
+```
+Refer to the description of the [CustomAlertmanager](cr.html#customalertmanager) Custom Resource and the [CustomAlertmanager](cr.html#alertmanager) for more information about the parameters
+
 ## How do I add an additional Alertmanager?
 
-Create a Custom Resource `CustomAlertmanager`, it can point to Alertmanager through the FQDN or Kubernetes service
+Create a Custom Resource `CustomAlertmanager` with type `External`, it can point to Alertmanager through the FQDN or Kubernetes service
```

Comment:
```suggestion
Create a custom resource `CustomAlertmanager` with type `External`, it can point to Alertmanager through the FQDN or Kubernetes service
```